Disable bandwidth-manager by default for new deployments

qmonnet opened the pull request:
Bandwidth manager is a beta feature in v1.9. Disable it by default for new deployments. Because it is already [disabled by default for the agent](https://github.com/cilium/cilium/blob/753231071fd29bca72b97e903f9337db4b17ee37/daemon/cmd/daemon_main.go#L536), this means we simply have to disable it on the Helm side.

The bandwidth manager is voluntarily re-enabled as a default for the `experimental-install.yaml` (through the Makefile), based on [feedback on Slack](https://cilium.slack.com/archives/C2B917YHE/p1602521560222000). File `quick-install.yaml` is updated.

Addresses one item for: https://github.com/cilium/cilium/issues/13371

pchaigno commented on install/kubernetes/cilium/values.yaml:
```diff
@@ -341,7 +341,7 @@ global:
   # bandwidthManager enables BBR and FQ + EDT + BPF based rate-limiting for
   # Pods that are annotated with kubernetes.io/egress-bandwidth.
-  bandwidthManager: true
+  bandwidthManager: false
```
We'll also need to update the K8sBandwidthTest tests to explicitly enable the bandwidth manager:
https://github.com/cilium/cilium/blob/d6ad56b02764f32db482c56e8dc41c42671a93eb/test/k8sT/Bandwidth.go#L146-L167

aanm reviewed (approved):
Changes LGTM except what Paul said. Also, we might need to wait until #13259 is merged

qmonnet committed:
K8sBandwidthTest: enforce usage of bandwidth manager when deploying

Make sure that tests for the bandwidth manager deploy with the manager
enabled, in case this were not performed by default.

Signed-off-by: Quentin Monnet <quentin@isovalent.com>

qmonnet committed:
EDT: disable bandwidth-manager for new deployments

Bandwidth manager is a beta feature as of this commit. Disable it by
default for new deployments.

Signed-off-by: Quentin Monnet <quentin@isovalent.com>

sayboras commented on install/kubernetes/cilium/values.yaml:
```diff
@@ -164,7 +164,7 @@ azure:
   # clientSecret: 00000000-0000-0000-0000-000000000000
 
 # TODO: Add documentation
```
If possible, maybe add docs here :thinking: , otherwise we can do it as part of #13496 as well :)

qmonnet replied:
The PR is about disabling the feature by default, and I intend to revert the related commit after the release is done. So I'd rather have the documentation added in another PR, if you don't mind.

sayboras reviewed (approved):
LGTM :100:
